resources/js/components/Search/Autocomplete.vue

```diff
@@ -1,14 +1,22 @@
 <script>
-import AisSearchBox from 'vue-instantsearch/vue2/es/src/components/SearchBox.vue.js'
-import AisIndex from 'vue-instantsearch/vue2/es/src/components/Index.js'
-
 import InstantSearchMixin from './InstantSearchMixin.vue'
 
+import InstantSearch from 'vue-instantsearch/vue2/es/src/components/InstantSearch'
+import Hits from 'vue-instantsearch/vue2/es/src/components/Hits.js'
+import Configure from 'vue-instantsearch/vue2/es/src/components/Configure.js'
+import highlight from 'vue-instantsearch/vue2/es/src/components/Highlight.vue.js'
+import SearchBox from 'vue-instantsearch/vue2/es/src/components/SearchBox.vue.js'
+import Index from 'vue-instantsearch/vue2/es/src/components/Index.js'
+
 export default {
     mixins: [InstantSearchMixin],
     components: {
-        'ais-search-box': AisSearchBox,
-        'ais-index': AisIndex,
+        InstantSearch,
+        Hits,
+        Configure,
+        highlight,
+        SearchBox,
+        Index,
```

Comment:
we must still USE these components in order to tell the bundling to load these at the same time as the autocomplete.
If we don't we re-introduce the horrible flashing as can be seen by the downloads
<img width="1260" alt="image" src="https://github.com/user-attachments/assets/edc8db3e-6bc9-4af7-b4dd-b51221eb6a7c" />

In it's state where we still import _and use_ the components the downloads look like this:
<img width="1237" alt="image" src="https://github.com/user-attachments/assets/19f01c75-29ce-45d6-8c5d-d357067fb6aa" />